mLRS/CommonTx/mbridge_interface.h

```diff
@@ -135,26 +135,24 @@ bool tMBridge::transmit_start(void)
 
 
 // is called in isr context
-// we can assume that there is at least one byte available
+// send in one chunk to help ensure it is transmitted with no gaps
 uint8_t tMBridge::send_serial(void)
 {
-    pin5_putc(0x00); // we can send anything we want which is not a command, send 0x00 so it is easy to recognize
-    for (uint8_t i = 0; i < MBRIDGE_M2R_SERIAL_PAYLOAD_LEN_MAX; i++) {
-        uint8_t c = serial_getc();
-        pin5_putc(c);
-        if (!serial_rx_available()) break;
+    uint8_t buf[MBRIDGE_M2R_SERIAL_PAYLOAD_LEN_MAX+1];
```

Comment:
same question/point ass the one before

Reply:
This is part of how I didn't break stm32